python/tvm/relay/frontend/tflite.py

```diff
@@ -224,6 +226,53 @@ def convert_reshape(self, op):
 
         return out
 
+    def _convert_resize(self, method, op):
+        """Generic method to Convert TFLite RESIZE operators"""
+        try:
+            from tflite.BuiltinOptions import BuiltinOptions
+            from tflite.Operator import Operator
+            from tflite.ResizeBilinearOptions import ResizeBilinearOptions
+            from tflite.ResizeNearestNeighborOptions import ResizeNearestNeighborOptions
+        except ImportError:
+            raise ImportError("The tflite package must be installed")
+
+        assert isinstance(op, Operator)
+        input_tensors = self.get_input_tensors(op)
+        assert len(input_tensors) == 2, "input tensors length should be 2"
+
+        # images, 4-D Tensor with shape NHWC.
+        input_tensor = input_tensors[0]
+        in_expr = self.get_expr(input_tensor.tensor_idx)
+
+        # size - 1-D int32 Tensor of 2 elements: new_height, new_width
+        size = self.get_tensor_value(input_tensors[1])
+        # convert numpy array to tuple
+        size = (size[0], size[1])
+
+        # Options - align_corners (bool)
+        if method == "BILINEAR":
+            assert op.BuiltinOptionsType() == BuiltinOptions.ResizeBilinearOptions
+            resize_options = ResizeBilinearOptions()
+        else:
+            assert op.BuiltinOptionsType() == BuiltinOptions.ResizeNearestNeighborOptions
+            resize_options = ResizeNearestNeighborOptions()
+
+        op_options = op.BuiltinOptions()
+        resize_options.Init(op_options.Bytes, op_options.Pos)
+        align_corners = resize_options.AlignCorners()
+
+        # Use layout NHWC
+        out = _op.image.resize(in_expr, size, "NHWC", method, align_corners)
```

Comment:
After previous, we could use `tuple(target_size)`

Reply:
fixed